node_configuration:
  topic_name: '/zed2/zed_node/right_raw/image_raw_color/compressed' #'/zed2i/zed_node/right/image_rect_color'
  node_name: 'showImagesFromBag'

